javapeople.yaml

```diff
@@ -234,6 +234,11 @@
   twitter: "FHHabermann"
   fediverse: "fhhabermann@fosstodon.org"
   github: "fh-ms"
+- name: "Foivos Zakkak"
+  twitter: "foivoszakkak"
+  fediverse: "zakkak@mastodon.online "
```

Comment:
The blank at the end results in an invalid handle.

Reply:
Fixed, thanks for the pointer.